aten/src/ATen/core/PythonFallbackKernel.cpp

```diff
@@ -64,47 +65,24 @@ void pythonFallback(const c10::OperatorHandle& op, c10::DispatchKeySet dispatch_
     return;
   }
 
-  // Otherwise, find a PyInterpreter on a Tensor
+  // Use global PyInterpreter instead of searching through tensors
   const auto& schema = op.schema();
   const auto num_arguments = schema.arguments().size();
-  // It is safe to dispatch on the very first Tensor with a pyobj_interpreter
-  // without checking the interpreters of any of the arguments, because when
-  // we actually run dispatch(), we will take out PyObjects in the context
-  // of that interpreter, and this will ensure that everyone is on the same
-  // interpreter.
   bool tensors_with_python_key_present = false;
-  c10::impl::PyInterpreter* interpreter = nullptr;
+  c10::impl::PyInterpreter* interpreter = c10::impl::getGlobalPyInterpreter();
+
+  // Still need to check if any tensors have the Python key
```

Comment:
Please update all the comments to not reflect what you are doing in the PR but the new state of the world.
"still" or "instead" shouldn't really be there.